examples/sql_bank/main.go

```diff
@@ -142,25 +155,44 @@ func verifyBank(db *sql.DB) {
 }
 
 func main() {
-	db, err := sql.Open("postgres", "dbname=postgres sslmode=disable")
-	if err != nil {
-		log.Fatal(err)
+	security.SetReadFileFn(securitytest.Asset)
+	serv := server.StartTestServer(nil)
```

Comment:
I'd like to do that but it makes the code all ugly. You have to declare all the variables. Its pretty harmless to leave this as is.